micromasters_import/management/commands/queries/007_import_transation.sql

```diff
@@ -0,0 +1,29 @@
+INSERT INTO public.ecommerce_transaction (
+    transaction_id,
+    amount,
+    data,
+    order_id,
+    transaction_type,
+    reason,
+    created_on,
+    updated_on
+)
+SELECT
+    mm_receipt.data->>'transaction_id',
+    mm_order.total_price_paid,
+    mm_receipt.data,
+    mo_order.id,
+    CASE
+         WHEN mm_order.status IN ('refunded', 'partially_refunded') THEN 'refund'
+         ELSE 'payment'
```

Comment:
You added the partially refunded state, so should this be mapped to that?

Reply:
This is different from the order state. This is the transaction state that only has two:
TRANSACTION_TYPE_REFUND = "refund"
TRANSACTION_TYPE_PAYMENT = "payment"
I thought the transaction type is just a way to identify if it's a refund or payment.  Should we add a third transaction type?

Reply:
Oh, no you're right. I need more coffee. Carry on.